Sources/SwiftlyCore/Platform.swift

```diff
@@ -268,32 +267,98 @@ extension Platform {
         }
     }
 
-    public func systemManagedBinary(_ cmd: String) throws -> String? {
+    // Install ourselves in the final location
+    public func installSwiftlyBin() throws {
+        // First, let's find out where we are.
+        let cmd = CommandLine.arguments[0]
+        let cmdAbsolute = if cmd.hasPrefix("/") {
+            cmd
+        } else {
+            ([FileManager.default.currentDirectoryPath] + (ProcessInfo.processInfo.environment["PATH"]?.components(separatedBy: ":") ?? [])).map {
+                $0 + "/" + cmd
+            }.filter {
+                FileManager.default.fileExists(atPath: $0)
+            }.first
+        }
+
+        // We couldn't find ourselves in the usual places. Assume that no installation is necessary
+        // since we were most likely invoked at SWIFTLY_BIN_DIR already.
```

Comment:
This would be a very strange case, but I suppose it could be possible in some way that I haven't considered. I expect this to be rare. The user is running `swiftly init` in some way, so they can continue to run it that way, we won't be copying the binary because we can't decide if it is a system binary or not.